apps/core-app/src/pages/Proposals.tsx

```diff
@@ -1,43 +1,130 @@
+import { ChangeEvent, MouseEvent, useEffect, useMemo, useState } from 'react';
 import styled from 'styled-components';
 import {
-  BiColumnLayout,
   Button,
-  Card,
-  widthQuery,
   Dialog,
   DialogTrigger,
   DialogContent,
+  SingleColumnLayout,
 } from '@daohaus/ui';
+import { statusFilter } from '@daohaus/dao-data';
 import { BsPlusLg } from 'react-icons/bs';
 
-import { useProposals } from '../contexts/DaoContext';
+import { defaultDaoData, useDao, useProposals } from '../contexts/DaoContext';
 import { NewProposalList } from '../components/NewProposalList';
 import { FORM } from '../legos/form';
-import { useMemo } from 'react';
+import SearchInput from '../components/SearchInput';
+import FilterDropdown from '../components/FilterDropdown';
+import { BaseProposalCard } from '../components/BaseProposalCard';
+import useDebounce from '../utils/debounceHook';
+import { PROPOSAL_STATUS } from '@daohaus/common-utilities';
 
-const LeftCard = styled(Card)`
+const ActionsContainer = styled.div`
   width: 100%;
-  min-width: 54rem;
-  max-width: 64rem;
-  @media ${widthQuery.md} {
-    max-width: 100%;
-    min-width: 0;
-  }
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 3rem;
+`;
+
+const SearchFilterContainer = styled.div`
+  display: flex;
+  gap: 2.1rem;
 `;
 
 export const VALID_NEW_PROPOSALS = [FORM.SIGNAL, FORM.SHARE_SWAP];
 
 export function Proposals() {
-  const { proposals } = useProposals();
+  const {
+    proposals,
+    setProposalsPaging,
+    proposalsNextPaging,
+    setProposalsFilter,
+    setProposals,
+  } = useProposals();
+  const { dao } = useDao();
+  const [searchTerm, setSearchTerm] = useState<string | ''>('');
```

Comment:
I think that empty quotes would be considered a string. So below should work:
```
  const [searchTerm, setSearchTerm] = useState<string>('');
```
or
```
const [searchTerm, setSearchTerm] = useState<string | null>(null);
```